pySpaceTraders/models/factions.py

```diff
@@ -1,41 +1,2 @@
-class Factions:
-    VALID = [
-        "COSMIC",
-        "VOID",
-        "GALACTIC",
-        "QUANTUM",
-        "DOMINION",
-        "ASTRO",
-        "CORSAIRS",
-        "OBSIDIAN",
-        "AEGIS",
-        "UNITED",
-        "SOLITARY",
-        "COBALT",
-        "OMEGA",
-        "ECHO",
-        "LORDS",
-        "CULT",
-        "ANCIENTS",
-        "SHADOW",
-        "ETHEREAL",
-    ]
-    COSMIC = "COSMIC"
-    VOID = "VOID"
-    GALACTIC = "GALACTIC"
-    QUANTUM = "QUANTUM"
-    DOMINION = "DOMINION"
-    ASTRO = "ASTRO"
-    CORSAIRS = "CORSAIRS"
-    OBSIDIAN = "OBSIDIAN"
-    AEGIS = "AEGIS"
-    UNITED = "UNITED"
-    SOLITARY = "SOLITARY"
-    COBALT = "COBALT"
-    OMEGA = "OMEGA"
-    ECHO = "ECHO"
-    LORDS = "LORDS"
-    CULT = "CULT"
-    ANCIENTS = "ANCIENTS"
-    SHADOW = "SHADOW"
-    ETHEREAL = "ETHEREAL"
+class Faction:
+    pass  # TODO
```

Comment:
stray TODO

Reply:
I just did the enums. I think @chatterchats is working on this?

Reply:
Indeedly Doo, It's in the other PR 👀